.github/workflows/linters.yml

```diff
@@ -0,0 +1,48 @@
+name: Linters
+
```

Comment:
- [ ] Please README file should be descriptive? [AKA: Microverse README template](https://github.com/microverseinc/readme-template) should be used.